### ISSUE TITLE
What are the required dimensions of `predict` and `target` for abCE loss?

### ISSUE THREAD

**Issue by JanineCHEN** [yassouali/CCT#67]
Thank you for the great work and shared repo!

I'm trying to do some customized training on my own dataset and I wanna use the proposed abCE Loss for training the encoder. Yet I'm facing some dimension mismatch issues while feeding my `predict` and `target` into the abCE loss function.

My `predict` and `target` both have a dimension of `BxCxHxW` and are leading to errors, even after I adjusted [this line](https://github.com/yassouali/CCT/blob/master/utils/losses.py#L203) by removing the unsqueeze(1) part.

It's a bit difficult to tell the input dimension requirement from the original code. I'm referring to this part in particular: https://github.com/yassouali/CCT/blob/master/utils/losses.py#L198

A bit of clarification about the input dimension of abCE loss would be very much appreciated! Thank you!

**Comment by yassouali** [comment]
Hi @JanineCHEN, sorry for the delay, is the issue solved?

**Comment by JanineCHEN** [comment]
Hi @yassouali , no worries! I have managed to solve the issue by replacing F.cross_entropy with F.binary_cross_entropy, it now can fit my customized dimension. : )
